modules/geo-layers/src/mvt-layer/mvt-layer.js

```diff
@@ -154,32 +162,57 @@ export default class MVTLayer extends TileLayer {
 
     const isWGS84 = this.context.viewport.resolution;
 
-    if (!isWGS84 && info.object) {
-      info.object = transformTileCoordsToWGS84(info.object, info.tile.bbox, this.context.viewport);
+    if (info.object) {
+      if (!isWGS84 && info.object) {
+        info.object = transformTileCoordsToWGS84(info.object, info.tile, this.context.viewport);
```

Comment:
Found an issue here while moving the function. It appears in 8.4.0-beta.2:

~info.tile~ info.tile.bbox

This breaks getting coordinates from geometries in picked objects.